src/main/resources/standardTestCases/confirm_preemptible/confirm_preemptible.wdl

```diff
@@ -0,0 +1,28 @@
+task check_preemption {
+  Int set_preemptible
+
+  command {
+    curl "http://metadata.google.internal/computeMetadata/v1/instance/scheduling/preemptible" -H "Metadata-Flavor: Google"
+  }
+
+  output {
+    String out = read_string(stdout())
+  }
+
+  runtime {
+    preemptible: "${set_preemptible}"
+    # ubuntu:trusty + curl
+    docker: "mcovarr/curly:v1"
```

Comment:
If there is I didn't find it.  I would much rather use something standard if that's available.